examples/floating-point-numbers/floating_point_numbers.bal

```diff
@@ -5,6 +5,11 @@ public function main() {
 
     int n = 5;
 
+    // Numeric literals can use `f` or `F` suffix for them to be interpreted as `float` values.
+    // (similarly, the `d` or `D` suffix can be used for `decimal`).
+    var f = 12345f;
+    io:println(f is float);
+
```

Comment:
Add this new scenario to consistent with decimal datatype, because decimal datatpe have the same kind of example.